tests/test_raw_building.py

```diff
@@ -19,6 +19,7 @@
 from typing import Any
 from unittest import mock
 
+import mypy.build
```

Comment:
I'm fine with this, but we would need to add `mypy` to the requirements file:
https://github.com/pylint-dev/astroid/blob/main/requirements_minimal.txt

It is now in the `_dev` file.

Reply:
Done :-)